waspc/src/Wasp/Psl/Parser/Argument.hs

```diff
@@ -0,0 +1,104 @@
+module Wasp.Psl.Parser.Argument
+  ( argument,
+    argumentList,
+    expression,
+    identifierStrParser,
+  )
+where
+
+import Text.Parsec
+  ( char,
+    choice,
+    optionMaybe,
+    try,
+  )
+import Text.Parsec.String (Parser)
+import qualified Wasp.Psl.Ast.Argument as Psl.Argument
+import Wasp.Psl.Parser.Common
+  ( brackets,
+    colon,
+    commaSep,
+    commaSep1,
+    float,
+    identifier,
+    integer,
+    parens,
+    stringLiteral,
+  )
+
+-- Parses attribute argument that ends with delimiter: , or ).
+-- Doesn't parse the delimiter.
+argument :: Parser Psl.Argument.Argument
+argument =
+  choice
+    [ try namedArg,
+      try unnamedArg
+    ]
+  where
+    namedArg :: Parser Psl.Argument.Argument
+    namedArg = do
+      name <- identifier
+      _ <- colon
+      Psl.Argument.ArgNamed name <$> expression
+
+    unnamedArg :: Parser Psl.Argument.Argument
+    unnamedArg = Psl.Argument.ArgUnnamed <$> expression
+
+argumentList :: Parser [Psl.Argument.Argument]
+argumentList = parens (commaSep (try argument))
+
+expression :: Parser Psl.Argument.Expression
```

Comment:
`argValue` became `expression` in this latest iteration to keep it in line with the Prisma grammar. It enabled us to parse the Prisma schema more confidently.